.eslintrc.yaml

```diff
@@ -0,0 +1 @@
+extends: screwdriver
```

Comment:
Since this plugin obeys the hapijs lint rules, maybe it should extend that eslint ruleset instead.
https://www.npmjs.com/package/eslint-config-hapi